File: leetcode_practice/010_Regular_Expression_Matching/solution.cpp
// 문제: 010 Regular Expression Matching
// 컴파일/실행: g++ solution.cpp -o sol.exe ; .\sol.exe
#include <iostream>
#include <string>
using namespace std;

class Solution {
public:
    bool isMatch(string s, string p) {
        // TODO: 여기에 작성하세요
        (void)s;
        (void)p;
        return false;
    }
};

int main() {
    Solution sol;
    // 직접 만든 입력
    cout << boolalpha << sol.isMatch("color", "c.*r") << endl;  // 기대값: true
    cout << boolalpha << sol.isMatch("code", "c*d") << endl;    // 기대값: false
    return 0;
}

File: leetcode_practice/010_Regular_Expression_Matching/solution.py
# 문제: 010 Regular Expression Matching
# 실행: py solution.py


class Solution:
    def isMatch(self, s: str, p: str) -> bool:
        # TODO: 여기에 작성하세요
        return False


if __name__ == "__main__":
    sol = Solution()
    # 직접 만든 입력
    print(sol.isMatch("color", "c.*r"))  # 기대값: True
    print(sol.isMatch("code", "c*d"))    # 기대값: False
